Lib/javascript/v8/javascriptinit.swg

```diff
@@ -70,14 +70,18 @@ extern "C"
 #if (NODE_MODULE_VERSION < 0x000C)
 void SWIGV8_INIT (v8::Handle<v8::Object> exports)
 #else
-void SWIGV8_INIT (v8::Handle<v8::Object> exports, v8::Handle<v8::Object> /*module*/)
+void SWIGV8_INIT (v8::Local<v8::Object> exports, v8::Local<v8::Object> /*module*/)
```

Comment:
This looks like an incompatible change to me, i.e. whichever versions of v8 that uses init function with 2 parameters of `v8::Handle` type would be broken by this change. I am not sure which versions these are and whether we care about them or not though.

Reply:
This code is from 2013. I believe 0x000C means Node.js v0.12. I have replaced everything with `SWIGV8_OBJECT` macro.